php-checks/src/main/java/org/sonar/php/checks/TooManyMethodsInClassCheck.java

```diff
@@ -90,6 +92,39 @@ private int getNumberOfMethods(ClassTree tree) {
     return nbMethod;
   }
 
+  private static boolean classIsDBEntity(ClassTree tree) {
+    boolean foriegnMethodFound = Boolean.FALSE;
+
+    for (ClassMemberTree classMember : tree.members()) {
+
+      if (!classMember.toString().contains("get") && !classMember.toString().contains("set")) {
```

Comment:
The specification expects that the methods start with "get" or "set". This logic would also allow method names like "calculateTar**get**Value".